xikolo-ios/ViewControllers/Courses/CourseDetailViewController.swift

```diff
@@ -83,28 +83,14 @@ class CourseDetailViewController: UIViewController {
         self.enrollmentButton.tintColor = UIColor.white
     }
     
-    func showEnrollmentDialog() {
-        let alertTitle = NSLocalizedString("enrollment.alert.title", comment: "title of enrollment alert")
-        let alertMessage = NSLocalizedString("enrollment.alert.message", comment: "message of enrollment alert")
-        let alert = UIAlertController(title: alertTitle, message: alertMessage, preferredStyle: .alert)
-
-        let enrollActionTitle = NSLocalizedString("enrollment.alert.enroll-action.title",
-                                                  comment: "title of enroll action of enrollment alert")
-        let enrollAction = UIAlertAction(title: enrollActionTitle, style: .default) { _ in
-            EnrollmentHelper.createEnrollment(for: self.course).flatMap {
-                CourseHelper.refreshCourses()
+    func createEnrollment() {
+        EnrollmentHelper.createEnrollment(for: self.course).flatMap {
+            CourseHelper.refreshCourses()
             }.onSuccess { _ in
                 if let parent = self.parent as? CourseDecisionViewController {
                     parent.decideContent()
                 }
```

Comment:
The code in lines 89-92 have to much indentation. Please fix this, then it's good to be merged.